Formula/solarus.rb

```diff
@@ -28,6 +29,8 @@ def install
       ENV.append_to_cflags "-I#{Formula["glm"].opt_include}"
       ENV.append_to_cflags "-I#{Formula["physfs"].opt_include}"
       system "cmake", "..",
+                      "-DCMAKE_INSTALL_RPATH=#{rpath}",
+                      "-DSOLARUS_ARCH=#{Hardware::CPU.arm? ? "arm64" : "x86_64"}",
```

Comment:
```suggestion
                      "-DSOLARUS_ARCH=#{Hardware::CPU.arch}",
```
I think this should be equivalent.